assets/blocks/course-outline/module-block/edit.js

```diff
@@ -29,24 +28,44 @@ import { useInsertLessonBlock } from './use-insert-lesson-block';
  * @param {string}   props.attributes.blockStyle  Selected block style.
```

Comment:
I think we need to update the props, like the `name`, `blockStyle`, `bordered` and the `borderColorValue`, right?

Reply:
Fixed in c84fdbcc32300a4a1e2ca89b8f9b2e72dbf98694